### ISSUE TITLE
Field's label can't be set locally to inform assistive technology users what to input (1.3.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux-phone-field#41]
### Expected behavior

- Input field has a label programatically associated to be provided to assistive technology users about what to input.
- Teams can associate the input to their context-specific label or provide a label.
- Default label doesn't include name of input and is only "Phone number".

As an example below, the an input label could need to be "Business phone number" and/or "Mobile phone number". 

### Actual behavior
The input has an aria-label="Phone input field". Aria-label is at the top of the hierarchy in the algorithm browsers use to determine an input's label. Users will always get this as the label and not whatever the label needs to be given what's being asked for (e.g., Business phone number).

### Impact
Assistive technology users won't be communicated correct input field label causing confusion and/or incorrect input.

**Comment by Blackbaud-TrevorBurch** [comment]
@Blackbaud-MattGregg overwriting the default works today (just verified this locally). However, I will get a PR up to update the default to "Phone number" instead of "Phone input field".

Just a quick reminder (just to be super clear) that when binding an aria label you must use `[attr.aria-label]` and not `[aria-label]`.

**Comment by Blackbaud-TrevorBurch** [comment]
@Blackbaud-MattGregg The PR for the updated default is #42

**Comment by Blackbaud-TrevorBurch** [comment]
In `4.0.1`
